app/index.js

```diff
@@ -9,98 +9,60 @@ class BlockstackGenerator extends Generator {
   constructor(args, opts) {
     // Calling the super constructor is important so our generator is correctly set up
     super(args, opts);
+    this.option('react', {
+      description: 'Generate a React app without prompting',
+      type: Boolean
+    });
+    this.option('plain', {
+      description: 'Generate a plain JavaScript app without prompting',
+      type: Boolean
+    });
+    this.option('vue', {
+      description: 'Generate a Vue app without prompting',
+      type: Boolean
+    });
   }
 
-  prompting() {
-    // Have Yeoman greet the user.
-    this.log(yosay(
-      'Welcome to the ' + chalk.red('Blockstack') + ' app generator!'
-    ));
+  async prompting() {
+    if (this.options.react) {
+      this.outputAppType = 'react';
+    } else if (this.options.plain) {
+      this.outputAppType = 'plain';
+    } else if (this.options.vue) {
+      this.outputAppType = 'vue';
+    } else {
 
-    var prompts = [{
-      type: 'confirm',
-      name: 'someAnswer',
-      message: 'Are you ready to build a Blockstack app?',
-      default: true
-    }];
+      // Have Yeoman greet the user.
+      this.log(yosay(
+        'Welcome to the ' + chalk.red('Blockstack') + ' app generator!'
+      ));
 
-    return this.prompt(prompts).then(function (props) {
-      // To access props later use this.props.someAnswer
-      this.props = props;
-    }.bind(this));
+      const answers = await this.prompt([{
+        type: 'list',
+        name: 'generator_type',
+        message: 'Select Blockstack app output type',
+        choices: [
```

Comment:
The `(Use arrow keys)` disappears after the one use. Is there any reason to make them disappear? 

`(Use arrow keys to select and press ENTER)`

Many of our app mining users are beginner programmers from other countries. It helps to use complete sentences and instructions and to leave the instructions in place.

Reply:
That is behavior and messaging hardcoded by what `yo` `prompt` uses: https://github.com/SBoudrias/Inquirer.js/blob/d3d632a7d94b3d719152c109f1865a8a9a2a2432/packages/inquirer/lib/prompts/list.js#L84-L86

One work around is to use raw index lists which lets that message be customized, but then forces a number-based UI (arrow keys _do_ still work):
![image](https://user-images.githubusercontent.com/1447546/64737601-5023b100-d4bb-11e9-95b8-91e2f3b56755.png)

Those appears to be our 2 options, I'm fine with either -- what do you think?

Reply:
Yeah, I suspected this might be the case.  Um, I think the number based thing is a lot more obvious than the keyboard input.  Let's go with that.

Reply:
Updated with that change ^